R/getGeneticInteractome.R

```diff
@@ -0,0 +1,42 @@
+# getGeneticInteractome.R
+#' @title getGeneticInteractome
+#' @author Nada Elnour, \email{nada.elnour@@mail.utoronto.ca}
+#'
+#' \code{getGeneticInteractome} augments systems with BioGRID genetic
+#' interaction annotations
+#'
+#' @param mySys The dataframe of PPI between system's components
```

Comment:
The @param and @return should match the format described here
https://github.com/hyginn/BCB420.2019.ESA/blob/master/PullRequestReviewNotes.md

Reply:
fixed